From 4a9e4ab0d12150805feca3012854de9fd4e5a721 Mon Sep 17 00:00:00 2001
From: Remi Collet <remi@remirepo.net>
Date: Fri, 6 Oct 2017 12:08:28 +0200
Subject: [PATCH] Fix #23 Failed to read session data with 7.1/7.2

---
 php7/memcache_session.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/php7/memcache_session.c b/php7/memcache_session.c
index dd2434a..45ba97d 100644
--- a/php7/memcache_session.c
+++ b/php7/memcache_session.c
@@ -289,7 +289,7 @@ PS_READ_FUNC(memcache)
 			ZVAL_NULL(&addresult);
 
 			/* third request fetches the data, data is only valid if either of the lock requests succeeded */
-			ZVAL_NULL(&dataresult);
+			ZVAL_EMPTY_STRING(&dataresult);
 
 			/* create requests */
 			if (php_mmc_session_read_request(pool, &zkey, lockparam, &addresult, dataparam, &lockrequest, &addrequest, &datarequest) != MMC_OK) {
